gettingStarted/sysElevenStackKickstart.yaml

```diff
@@ -156,27 +157,32 @@ resources:
               /bin/chown syseleven:syseleven $bin/terraform
               /bin/chown -R syseleven:syseleven $bin/
               if [ -f $tmp ]; then rm $tmp; fi
-              if [ -x $bin/terraform ]; then exit 0 && echo "# Finished terraform setup"; else (echo "Error while installing terraform"; exit 99); fi
+              if [ -x $bin/terraform ]; then (echo "# Finished terraform setup"; exit 0) else (echo "# Error while installing terraform"; exit 99); fi
            path: /root/terra-install.sh
            owner: root
            permissions: 0700
-
+         - content: |
+              #!/usr/bin/env bash
+              echo "# Started openstack client setup"
+              until id syseleven > /dev/null; do sleep 2; done
```

Comment:
why wait for the syseleven user to exist?

Reply:
This can be removed in future. Not required.